src/components/layouts/DefaultLayout.tsx

```diff
@@ -4,83 +4,30 @@ import Div100vh from 'react-div-100vh';
 import SideBar from './SideBar';
 import BackHeader from '../headers/BackHeader';
 import LogoHeader from '../headers/LogoHeader';
+import ScrollableContentLayout from './ScrollableContentLayout';
+import ContentLayout from './ContentLayout';
 
-const DefaultLayout = ({ children, onScroll = () => {} }: any) => {
+const DefaultLayout = ({ children, onScroll = () => {}, scrollable = true }: any) => {
   const isMobile = useWindowSize(device.mobileL);
   const currentRoute = useGetCurrentRoute();
   return (
     <Container>
       {!isMobile && <SideBar />}
-      <ScrollableContainer onScroll={onScroll}>
-        <InnerContainer>
-          {currentRoute?.back ? <BackHeader /> : <LogoHeader />}
-          <Content>
-            {currentRoute?.title && <Title>{currentRoute?.title}</Title>}
-            {currentRoute?.title && <Subtitle>{currentRoute?.title}</Subtitle>}
-            {children}
-          </Content>
-        </InnerContainer>
-      </ScrollableContainer>
+      {scrollable ? (
+        <ScrollableContentLayout currentRoute={currentRoute} onScroll={onScroll}>
```

Comment:
onScroll, atrodo is niekur nepassinama, gal nuo seno kazkokio kodo like, tai gal istrinti reiktu prie to pacio?

Reply:
Plius  nesu tikras ar tikrai reikia 2 layoutu ?  gal gana kazkur padaryt tiesiog `overflow-y: auto;` ir viskas ?